src/components/BlogList.jsx

```diff
@@ -21,15 +21,17 @@ function BlogList() {
         onPageChange={updatePage}
         onPageSizeOptionChange={updateRowsPerPage}
       />
-      {currentPaginationData.map((blog) => (
-        <BlogPost
-          key={blog.id}
-          author={blog.author}
-          title={blog.title}
-          excerpt={blog.excerpt}
-          featureImage={blog.image}
-        />
-      ))}
+      <ul aria-label="blog list">
```

Comment:
I made some accessibility improvements in this page because it is easier to test screen-reader-friendly page. `@testing-library` also recommends this practice ([doc](https://testing-library.com/docs/queries/about#priority)).

Some references I followed:
- https://www.a11ymatters.com/pattern/pagination/
- https://tink.uk/using-the-aria-current-attribute/

Reply:
Should we add some comments to not remove this aria-label in this place and others?